Makefile

```diff
@@ -42,5 +42,21 @@ get-server-code:
 	echo "Getting model_server code"
 	git clone https://github.com/gsilva2016/model_server 
 
-clean-all: clean clean-simulator
+clean-ovms-client:
+	if [ -z $$(docker ps  --filter="name=ovms-client" -q -a) ]; then\
```

Comment:
please use local variable to store the result of docker ps and then use that to do if-else and clean up logic